feat(content): HR: make A11more engaging and align with noted narrative needs (#7838)
endless-sky/endless-sky: data/hai/hai reveal 4 middle.txt

	on offer
		event "hai and human ships at wormhole"
		log `Teeneep has been rejected by the Elders again, but has set in a motion a plan which has triggered an election instead. This snap election has closed the border and risked creating a diplomatic incident.`

> 		log "People" "Aimet Taepip" `An Elder of advanced age and poor eyesight. He is the "master navigator" of the Hai political system, being the first to improve on a system in several generations. He commands widespread respect and possesses enormous clout in the Elder council.`

Reviewer: ```suggestion
		log "People" "Aimet Taepip" `An Elder of advanced age and poor eyesight. He is the "master navigator" of the Hai political system, being the first to improve on a system in several generations. He commands widespread respect and possesses enormous clout in the Council of Elders.`
```

		conversation
			`Upon your return, Teeneep lodges a formal demand for another audience with the Elders, including the context that she has acquired support from the Unfettered with appropriate proof. Once sent, Sayari steps out for a while to talk to someone and ensure that the demand is being processed with appropriate speed, leaving you and Teeneep alone in her office.`
			`	Teeneep spends a few moments more doing some other things at her terminal before finally sitting back with an expression of quiet determination. In the quiet she turns to you from her seat and says, "Well, all the pieces are set, now I suppose all that remains is to play out the moves."`
			label teeneepquestions
			choice

> 				`	"What will happen now then?"`

Reviewer: ```suggestion
				`	"What will happen now, then?"`
```

Author: Technically correct, but I feel like it breaks up the flow when reading it as spoken text.

Reviewer: Then we can keep it in.

					to display
						not "Hai Reveal: what will happen now?"
				`	"Do you think the Elders will listen?"`
					goto listen
					to display
						not "Hai Reveal: will the elders listen?"
				`	"Will there truly be a war between Hai and Unfettered?"`
					goto war
					to display
						or
							has "Hai Reveal: what will happen now?"
							has "Hai Reveal: will the elders listen?"
						not "Hai Reveal: will there be war?"
				`	"What are the other options?"`
					goto options
					to display
						has "Hai Reveal: will there be war?"
				`	"Who is Aimet?"`
					goto aimet
					to display
						has "Hai Reveal: will there be war?"
			action
				set "Hai Reveal: what will happen now?"

> 			`	She shrugs. "It is honestly quite hard to say. I struggle to believe that the Elders could be so unwilling to act that they would risk escalating the standoff with the Unfettered into an actual war. Our brethren are no longer merely a disaffected rabble of veterans camping on the one world that supported radical change. They are a truly empowered movement with consolidated support at home and sympathizers amongst the Hai at large."`

Reviewer: ```suggestion
			`	She shrugs. "It is honestly quite hard to say. I struggle to believe that the Elders could be so unwilling to act that they would risk escalating the standoff with the Unfettered into a real war. Our brethren are no longer merely a disaffected rabble of veterans, camping on the one world that supported radical change. They are a truly empowered movement with consolidated support at home and sympathizers amongst the Hai at large."`
```
I don't know if you agree with this change, but I like to think that their war has been fake all along, even though the fighting was real.

				goto teeneepquestions
			label listen
			action
				set "Hai Reveal: will the elders listen?"
			`	She sighs. "I wish to believe so, but part of me I can no longer ignore does not think it actually likely. Still, it is essential that I try with everything I have to make them do so. Otherwise there will be change, and we can be an unpredictable people in the face of that."`
				goto teeneepquestions
			label war
			action
				set "Hai Reveal: will there be war?"

> 			`	She frowns and gently shakes her head. "No, or at least, not yet. The willingness of the Unfettered to threaten war is essential to all of the possible paths from here. Even if the Elders refuse to take action though, there are more steps between this meeting and that outcome. The options are not yet exhausted, though they are rapidly dwindling. With luck though, Aimet's influence might tip the situation in our favor."`

Reviewer: ```suggestion
			`	She frowns and gently shakes her head. "No, or at least not yet. The willingness of the Unfettered to threaten war is essential to all of the possible paths from here. Even if the Elders refuse to take action, there are more steps between this meeting and that outcome. The options are not yet exhausted, though they are rapidly dwindling. With luck, though, Aimet's influence might tip the situation in our favor."`
```

				goto teeneepquestions
			label options

> 			`	"Ah, well those are difficult to explain. Since the options worth using will depend on exactly what happens in the meeting. One thing Sayari was very right to remind me off before is that there are normal channels and processes to follow, and those must be followed. We're an old people <last>, and not always a very patient people. We've forged a system of government with systems and processes to allow almost any internal situation to be handled without resorting to base hysteria, and to actively punish and isolate those who try and subvert those processes. It is a difficult pill to swallow sometimes, but it works, and it has ensured stability since before humans learned to bash rocks together."`

Reviewer: ```suggestion
			`	"Ah, well those are difficult to explain, since the options worth using will depend on exactly what happens in the meeting. One thing Sayari was very right to remind me of before is that there are normal channels and processes to follow, and those must be followed. We're an old people, Captain <last>, and not always a very patient people. We've forged a system of government with systems and processes to allow almost any internal situation to be handled without resorting to base hysteria, and to actively punish and isolate those who try and subvert those processes. It is a difficult pill to swallow sometimes, but it works, and it has ensured stability since before humans learned to bash rocks together."`
```

> 			`	Her mouth quirks in a sly little smile, "Aimet has always been the master navigator of those systems. He's responsible for improving one of them for the first time in two thousand years." She frowns. "It's actually very odd that he wasn't chosen for the Triad last time we were here."`

Reviewer: ```suggestion
			`	Her mouth quirks in a sly little smile. "Aimet has always been the master navigator of those systems. He's responsible for improving one of them for the first time in two thousand years." She frowns. "It's actually very odd that he wasn't chosen for the Triad last time we were here."`
```

				goto return
			label aimet

> 			`	"He is the," she pauses, "I don't think it translates quite right. You have a phenomenon in your language where you amalgamate words sometimes, like an engineer who works specifically on solutions to problems may be colloquially referred to as a 'solutioneer'. We have a similar thing with our words for political systems and navigator. It's a title he has earned."`

Reviewer: ```suggestion
			`	"He is the..." she pauses to look up aimlessly, "I don't think it translates quite right. You have a phenomenon in your language where you amalgamate words sometimes, like an engineer who works specifically on solutions to problems might be called a 'solutioneer'. We have a similar thing with our words for political systems and navigator. It's a title he has earned."`
```
Fewer and simpler words is better for me when it says the same thing with the same detail.

> 			`	She spreads her arms, 'palms' up, in a human gesture indicating a loss for words, "Our system of government has systems and processes to allow almost any internal situation to be handled without resorting to base hysteria, and to actively punish and isolate those who try and subvert those processes. It is a difficult pill to swallow sometimes, but it works, and it has ensured stability since before humans learned to bash rocks together. Aimet is the master navigator of those systems. He's responsible for improving one of them for the first time in two thousand years."`

Reviewer: ```suggestion
			`	She spreads her arms, 'palms' up, in a human gesture indicating a loss for words. "Our system of government has systems and processes to allow almost any internal situation to be handled without resorting to base hysteria, and to actively punish and isolate those who try and subvert those processes. It is a difficult pill to swallow sometimes, but it works, and it has ensured stability since before humans learned to bash rocks together. Aimet is the master navigator of those systems. He's responsible for improving one of them for the first time in two thousand years."`
```

			
			label return
			`	Teeneep's console beeps a moment prior to Sayari's slightly breathless return. A message, entirely in your language, is displayed on the screen; it announces an emergency meeting of the full Elder Council in under an hour.`

> 			`	Sayari steps across to Teeneep and glances at the message, "I needn't have left this room. We must prepare quickly, we're going to have an escort..."`
> 			`	Teeneep's expression darkens but she quickly ensures she has what she needs before a group of council security staff arrive to take you to the chambers. This is not the first time you've seen the room, but it is the first time you've seen it full. There is a small desk and seat at the front of the central circle, opposite the entry point, which was not occupied when you entered here for dealing with Scar's Legion but now hosts a very serious Hai bearing unusual marks of office. Sayari sees your line of vision and nudges you, conveying in a whisper, "That's the moderator. They're only required when more than two thirds of the council is in attendance, and when there's a debate or petition to address."`

Reviewer: ```suggestion
			`	Sayari steps across to Teeneep and glances at the message. "I needn't have left this room. We must prepare quickly, we're going to have an escort..."`
			`	Teeneep's expression darkens, but she quickly ensures she has what she needs before a group of Council security staff arrive to take you to the chambers. This is not the first time you've seen the room, but it is the first time you've seen it full. There is a small desk and seat at the front of the central circle, opposite the entry point, which was not occupied when you entered here for dealing with Scar's Legion but now hosts a very serious Hai bearing unusual marks of office. Sayari sees your line of vision and nudges you, conveying in a whisper, "That's the moderator. They're only required when more than two thirds of the council is in attendance, and when there's a debate or petition to address."`
```

> 			`	She glances around the room and raises an eyebrow. "This is a closed session, which is quite unusual for a full Council Assembly. This does not bode well."`
> 			`	As the security take their place the moderator walks over to address you while the Elders are still filtering in and muttering to each other. "Now, before we begin I would like to confirm. I understand you are able to act as a witness, and confirm that the threat carried from the Unfettered was indeed real. Is this correct?"`

Reviewer: ```suggestion
			`	She glances around the room and raises an eyebrow. "This is a closed session, which is quite unusual for a full Council Assembly. This does not bode well."`
			`	As the security staff take their place, the moderator walks over to address you while the Elders are still filtering in and muttering to each other. "Now, before we begin, I would like to confirm. I understand you are able to act as a witness, and confirm that the threat carried from the Unfettered was indeed real. Is this correct?"`
```
Bloody hell I hate the word 'indeed' but I can't change it.

			choice
				`	"That is correct."`
					goto correct
				`	"I can also testify to the human position on the matter."`
			`	The moderator pulls a strange face and says, "Right, well, I doubt that will be necessary.`
			label correct

> 			`	"Thank you Captain. Since you have become involved as a witness, protocol dictates that the session will be held in your language, though for the sake of order normal council directions will still be in Hai. We'll begin in a moment."`

Reviewer: ```suggestion
			`	"Thank you, Captain. Since you have become involved as a witness, protocol dictates that the session will be held in your language, though for the sake of order normal council directions will still be in Hai. We will begin in a moment."`
```

			`	After everyone is set in place, the moderator abruptly starts. "Teeneep has provided a confession about her visit to the Unfettered, incitement to war, involvement of outside government in internal Hai matters, and other concerns. She has justified this by an argument of necessity, that it is essential for action to be taken to produce both a peaceful outcome for the ongoing crisis with humanity and to heal a hole in the heart of our society."`

> 			`	When the moderator refers to 'healing the hole', it is spoken with an emphasis that implies heavy quotation, and you can hear the muttering of translations and scoffs rippling out among the elders in the room.`

Reviewer: ```suggestion
			`	When the moderator refers to 'healing the hole', it is spoken with an emphasis that implies heavy quotation, and you can hear the muttering of translations and scoffs rippling out among the Elders in the room.`
```

			`	Sayari hisses with distaste and quietly says, "The framing of this is terrible, someone has already decided what outcome they'd like to get and has got in on this process early."`
			`	The moderator calls something out in Hai which ends with 'Teeneep' and Sayari translates, "The Council hears Teeneep."`

> 			`	Not visibly disheartened by the situation Teeneep steps into the middle and begins. She makes her case thoroughly and passionately, but with a clearly well-practiced presentation structure, ensuring that the full background for the situation is laid out for those present. Some elders make snide comments at various points but the moderator shuts them down, and those most skeptical settle for being merely disengaged. When she presents the Unfettered's position as an offer of assistance there is a smattering of largely disbelieving sounds, though when she speaks of their willingness to force the matter the tone of the conversation gets remarkably more tense. "If you don't act, they will. Whether that's with you, or through you," she finishes.`

Reviewer: ```suggestion
			`	Not visibly disheartened by the situation, Teeneep steps into the middle and begins. She makes her case thoroughly and passionately, but with a clearly well-practiced presentation structure, ensuring that the full background for the situation is laid out for those present. Some Elders make snide comments at various points, but the moderator shuts them down and those most skeptical settle for being merely disengaged. When she presents the Unfettered's position as an offer of assistance there is a smattering of largely disbelieving sounds, though when she speaks of their willingness to force the matter the tone of the conversation gets remarkably more tense. "If you don't act, they will. Whether that's with you, or through you," she finishes.`
```
You gotta love commas. For the second sentence, I don't know which way is better or whether it's correct at all.

			`	One of the more stubborn Elders stands up, heralded by the moderator. "The Unfettered have made serious attempts to push through at Wah Ki before, to no avail. More recently, they've been more trouble again, and again we have organized more than enough security to keep their disruption contained, even with the current human crisis. This threat is ridiculous, and the Unfettered would not dare, not while they are so dependent on our food and medical supplies these past years. They would most certainly sacrifice many thousands of lives in the pursuit of misguided justice for a mere handful."`

> 			`	"You are blind. Blind and old and foolish," snaps Teeneep, but another elder has leaped to their feet and the moderator cuts her off with a rebuke that you don't need translated to know what it means.`

Reviewer: ```suggestion
			`	"You are blind. Blind and old and foolish," snaps Teeneep, but another Elder has leaped to their feet and the moderator cuts her off with a rebuke that you don't need translated to know what it means.`
```

> 			`	After being properly heralded, the new elder addresses you. "I understand Teeneep has enlisted you as her gopher. You have my sympathy. I have no doubt that your continued involvement means that you accept her conspiracy theories that many of the pirate activities, specifically relating to kidnapping and people smuggling, are a coordinated effort. So I will not waste time retreading your position on points she has just laid out. I must ask you of this recent interaction though. Tell me, how would you characterize Teeneep's meeting with the Unfettered Choot'k: constructive, aggressive, or something else?"`

Reviewer: ```suggestion
			`	After being properly heralded, the new Elder addresses you. "I understand Teeneep has enlisted you as her gopher. You have my sympathy. I have no doubt that your continued involvement means that you accept her conspiracy theories that many of the pirate activities, specifically relating to kidnapping and people smuggling, are a coordinated effort. So I will not waste time retreading your position on points she has just laid out. I must ask you of this recent interaction, though. Tell me, how would you characterize Teeneep's meeting with the Unfettered Choot'k: constructive, aggressive, or something else?"`
```

			choice
				`	"It was a constructive discussion."`
					goto constructive
				`	"I witnessed the test of two wills."`
					goto willed
				`	"The meeting was poignant."`
					goto poignant
				`	"Very aggressive and confronting."`
			action
				set haiignorance

> 			`	You hear many sighs, and several people shrug, but nobody seems surprised, just disappointed. "They grow ever more brazen in their desire for change," says the Elder. "Thank you Captain <last>."`

Reviewer: ```suggestion
			`	You hear many sighs, and several Elders shrug, but nobody seems surprised, just disappointed. "They grow ever more brazen in their desire for change," says the Elder. "Thank you, Captain <last>."`
```

				goto next
			label willed
			action
				set haiignorance

> 			`	Many elders chuckle around the room. The Elder says, "Ah, our pair of old rivals, divided by Cloudfire. The embers are not yet quenched then." At this, Teeneep rolls her eyes and the Elder departs.`

Reviewer: ```suggestion
			`	Many elders chuckle around the room. The Elder says, "Ah, our pair of old rivals, divided by Cloudfire. The embers have not yet quenched then." At this, Teeneep rolls her eyes and the Elder departs.`
```
Not sure about this, if it's some quote or something.

Reviewer: Yeah, I would like to know why, though.

Author: Embers do not quench themselves, it's a thing done to them. Therefore it cannot be "have" because that would be for processes that occur without interference.

				goto next
			label poignant
			action
				set haiinsight
			`	There is a quiet shuffling as some of the elders sit up a little straighter. The Elder says, "A common ground between rivals, a fearsome thing indeed."`
				goto next
			label constructive
			action
				set haiinsight

> 			`	A roomful of laughter engulfs your words, but many Elders appear offended by that mockery. The Elder who questioned you explains. "Elders and Unfettered have different views of what constitutes a 'constructive discussion.' You lack the biases of a Hai, so if you say it was constructive, I believe you." The tone of the room changes after that, as the laughter ends. Most Elders appear to agree with your assessment, and many look ashamed.`

Reviewer: ```suggestion
			`	A roomful of laughter engulfs your words, but many Elders appear offended by their peers' mockery. The Elder who questioned you explains, "The Elders and the Unfettered have different views of what constitutes a 'constructive discussion.' You lack the biases of a Hai, and if you say it was constructive, I believe you." The tone of the room changes after that, as the laughter ends. Most Elders appear to agree with your assessment, and many who laughed look ashamed.`
```

Author: ```suggestion
			`	A roomful of laughter engulfs your words, but many Elders appear offended by their peers' mockery. The Elder who questioned you explains, "The Elders and the Unfettered have different views of what constitutes a 'constructive discussion.' You lack the biases of a Hai. If you say it was constructive, I believe you." The tone of the room changes after that, as the laughter ends. Most Elders appear to agree with your assessment, and many who laughed look ashamed.`
```
This is just your suggestion but where you changed so to and I think it's actually much more proper if that's broken into two sentences there. Has a little more gravitas, which would be the aim.

			label next
			`	Sayari gives you a meaningful look and whispers, "A lot of them paid attention to that..."`

> 			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are all of you cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`

Reviewer: ```suggestion
			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "All of you are cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`
```
I need this Gelo Appanoh backstory, it keeps coming up whenever we redo this and I want to know about it. Really does sound interesting.

Author: Hehe it'll come.

For this one though "You are all of you cowards" is an archaic speech construction which I think feels appropriate for one of the oldest elders to have adopted. I'd be disinclined to change it unless the meaning is unclear.

Reviewer: I had to read it twice, I wasn't sure if it was "You are, all of you, cowards".

Author: Ah yes, those commas would be appropriate.

> 			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are all of you cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`

Reviewer: ```suggestion
			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`
```

> 			`	As the crescendo of her impassioned statement crests to its conclusion there is a growing rumble of ascent and the room is suddenly filled with the angry cries of those both emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen elders jump to their feet but the cacophony is not so easily restrained. At your side Sayari's expression is ashen but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`

Reviewer: ```suggestion
			`	As the crescendo of her impassioned statement crests to its conclusion, there is a growing rumble of ascent and the room is suddenly filled with the angry cries of those both emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen Elders jump to their feet, but the cacophony is not so easily restrained. At your side, Sayari's expression is ashen, but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`
```

> 			`	There is a shift in the sound of the room and your attention is drawn to one Hai in particular, a particularly elderly man who struggles to his feet in a series of pained stages. The effect on the assembly is profound though. A ripple of silence flows outward from his position and elders all around promptly drop back into their seats as it reaches them, leaving a scant handful who were too slow to react to have their final words trail off awkwardly. Only the last speaker remains on her feet, the moderately weakly announcing what you know to be, "The council hears Aimet."`

Reviewer: ```suggestion
			`	There is a sudden shift in the sound of the room. Your attention is drawn to one Hai in particular, a particularly elderly man who struggles to his feet in a series of pained stages. The effect on the assembly is profound, though: a ripple of silence flows outward from his position and Elders all around promptly drop back into their seats as it reaches them, leaving a scant handful, who were too slow to react, to have their final words trail off awkwardly. Only the last speaker remains on her feet, while the moderator weakly announces, "The Council hears Aimet."`
```
Interesting to see. This suggestion can be changed, I expect people to ask about it.

Author: I'm not a fan of how this one changes up the pacing, and I think the use of more advanced punctuation detracts from the reader's focus on the event.
Also the moderator is speaking Hai, you've just heard the phrase enough now that you know what it means.

> 			`	There is a shift in the sound of the room and your attention is drawn to one Hai in particular, a particularly elderly man who struggles to his feet in a series of pained stages. The effect on the assembly is profound though. A ripple of silence flows outward from his position and elders all around promptly drop back into their seats as it reaches them, leaving a scant handful who were too slow to react to have their final words trail off awkwardly. Only the last speaker remains on her feet, the moderately weakly announcing what you know to be, "The council hears Aimet."`

Reviewer: ```suggestion
			`	There is a shift in the sound of the room and your attention is drawn to one Hai in particular, a particularly elderly man who struggles to his feet in a series of pained stages. The effect on the assembly is profound, though. A ripple of silence flows outward from his position, and Elders all around promptly drop back into their seats as it reaches them, leaving a scant handful who were too slow to react to have their final words trail off awkwardly. Only the last speaker remains on her feet, the moderator weakly announcing in Hai what you know to be, "The council hears Aimet."`
```
I don't like the pacing, but I'll leave that up to you.

> 			`	"Security through secrecy..." He coughs. "You throw that in there as if it were always so Gemnali but, unlike you, I was there when it was added to the mandate of this chamber."`

Reviewer: ```suggestion
			`	"Security through secrecy..." He coughs. "You throw that in there as if it were always so, Gemnali, but unlike you, I was there when it was added to the mandate of this chamber."`
```

Reviewer: I think it needs the commas. He is meant to speak slowly?

> 			`	He peers around the room and you realize he is blind, or nearly so. "Freedom may be achieved through peace, but peace must not be maintained at the cost of freedom. Order may not be pursued at the cost of peace, nor security at the cost of order. Always this mandate must be considered in both directions. To censure this woman, to resist action on this matter. You would have us pursue secrecy at the cost of security."`

Reviewer: ```suggestion
			`	He blankly peers around the room, and you realize he is blind, or nearly so. "Freedom may be achieved through peace, but peace must not be maintained at the cost of freedom. Order may not be pursued at the cost of peace, nor security at the cost of order. Always this mandate must be considered in both directions. To censure this woman, to resist action on this matter. You would have us pursue secrecy at the cost of security."`
```

> 			`	He peers around the room and you realize he is blind, or nearly so. "Freedom may be achieved through peace, but peace must not be maintained at the cost of freedom. Order may not be pursued at the cost of peace, nor security at the cost of order. Always this mandate must be considered in both directions. To censure this woman, to resist action on this matter. You would have us pursue secrecy at the cost of security."`

Reviewer 2: ```suggestion
			`	He peers around the room, and you realize he is blind, or nearly so. "Freedom may be achieved through peace, but peace must not be maintained at the cost of freedom. Order may not be pursued at the cost of peace, nor security at the cost of order. Always this mandate must be considered in both directions. To censure this woman, to resist action on this matter. You would have us pursue secrecy at the cost of security."`
```

> 			`	He coughs again it echoes in the silence of nearly a hundred Hai holding their breath. When he continues his voice has grown stronger and more cutting. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`

Reviewer: ```suggestion
			`	He coughs again, echoing in the silence of nearly a hundred Hai holding their breath. When he continues, his voice grows stronger and sharper. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman, who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`
```

Author: I think `He coughs again, and it echoes` would be better still, but probably a good catch on the switch to perfect present tense halfway there.
Not a fan of the comma after woman though.

> 			`	He coughs again it echoes in the silence of nearly a hundred Hai holding their breath. When he continues his voice has grown stronger and more cutting. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`

Reviewer: ```suggestion
			`	He coughs again, and it echoes in the silence of nearly a hundred Hai holding their breath. When he continues, his voice grows stronger and sharper. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`
```

			`	The silence stretches with all eyes on Gemnali. She clenches her fists, shaking with barely suppressed emotion, before collapsing back into her seat as if physically struck.`
			`	Whatever spell had held the room is immediately broken and a susurrus of murmuring rises to fill its space. Within the noise a certain phrase starts to be repeated multiple times, drawing an expression of shock from Sayari.`
			choice
				`	"What are they saying?"`
				`	"What just happened?"`

> 			`	She glances at you briefly. "They are saying; Veeheylur and Misanaya fight over Eyrisen." Her tone carries with it a sense of awe and despite the growing volume in the room she leans in close to you and continues in scarcely more than a whisper. "The mythos has been invoked."`

Reviewer: ```suggestion
			`	She glances at you briefly. "They are saying: Veeheylur and Misanaya fight over Eyrisen." Her tone carries with it a sense of awe, and despite the growing volume in the room, she leans in close to you and continues in scarcely more than a whisper: "The mythos has been invoked."`
```

Author: *squints* 
I'm quite sure that second colon could be replaced with a better option.

> 			`	She glances at you briefly. "They are saying; Veeheylur and Misanaya fight over Eyrisen." Her tone carries with it a sense of awe and despite the growing volume in the room she leans in close to you and continues in scarcely more than a whisper. "The mythos has been invoked."`

Reviewer: ```suggestion
			`	She glances at you briefly. "They are saying: Veeheylur and Misanaya fight over Eyrisen." Her tone carries with it a sense of awe, and despite the growing volume in the room she leans in close to you and continues in scarcely more than a whisper, "The mythos has been invoked."`
```

> 			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator who struggles to compose himself suitably. "I think the council has heard enough. The petitioner is entitled to a closing statement unless there is objection?"`

Reviewer: ```suggestion
			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator who is struggling to compose himself suitably. "I think the Council has heard enough. The petitioner is entitled to a closing statement, unless there is objection?"`
```
I'm no good at tenses, but this sounds better to me. I need an opinion from EjoThims.

Reviewer 2: Both are correct, with very little distinction in meaning, so comes down to preference there.

Rest of these changes are necessary, though.

Author: ```suggestion
			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator who struggles to compose himself suitably. "I think the Council has heard enough. The petitioner is entitled to a closing statement, unless there is objection?"`
```

> 			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator who struggles to compose himself suitably. "I think the council has heard enough. The petitioner is entitled to a closing statement unless there is objection?"`

Reviewer 2: ```suggestion
			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator, who struggles to compose himself suitably. "I think the council has heard enough. The petitioner is entitled to a closing statement unless there is objection?"`
```

Also needs a comma here.

> 			`	None of those assembled who might object seem to have recovered their composure enough to stand up quickly and the moderator, perhaps sensing a politic moment to intervene, quickly announces the closing of the interrogation and calls the council to hear Teenep again.`

Reviewer: ```suggestion
			`	None of those assembled who might object seem to have recovered their composure enough to stand up quickly, and the moderator, perhaps sensing a politic moment to intervene, quickly announces the closing of the interrogation and calls the Council to hear Teeneep again.`
```

> 			`	Teeneep quickly plasters her mask of calm back on as best she is able and says, "Thank you all for your ear. You are those we trust to decide, but know this. If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`

Reviewer: ```suggestion
			`	Teeneep quickly plasters back on her mask of calm as best she is able to and says, "Thank you all for your ears. You are those we trust to decide, but know this: If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed, no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`
```
From Ireland to Cathay.

> 			`	One Elder cries out, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!"`
> 			`	Before any further exchange can occur though the moderator bellows for security to escort you all from the session and shouts down the offending Elder for impropriety. The two remain engaged in a lively shouting match as you depart, the echoes following you until the chamber doors are shut.`

Reviewer: ```suggestion
			`	One Elder cries out, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!"`
			`	Before any further exchange can occur, though, the moderator bellows for security to escort you all from the session and shouts down the offending Elder for impropriety. The two remain engaged in a lively shouting match as you depart, the echoes following you until the chamber doors are shut.`
```

			`	Teeneep spends a few moments more doing some other things at her terminal before finally sitting back with an expression of quiet determination. In the quiet she turns to you from her seat and says, "Well, all the pieces are set, now I suppose all that remains is to play out the moves."`
			label teeneepquestions
			choice
				`	"What will happen now then?"`
					to display
						not "Hai Reveal: what will happen now?"
				`	"Do you think the Elders will listen?"`
					goto listen
					to display
						not "Hai Reveal: will the elders listen?"
				`	"Will there truly be a war between Hai and Unfettered?"`
					goto war
					to display
						or
							has "Hai Reveal: what will happen now?"
							has "Hai Reveal: will the elders listen?"
						not "Hai Reveal: will there be war?"
				`	"What are the other options?"`
					goto options
					to display
						has "Hai Reveal: will there be war?"
				`	"Who is Aimet?"`
					goto aimet
					to display
						has "Hai Reveal: will there be war?"
			action
				set "Hai Reveal: what will happen now?"

> 			`	She shrugs. "It is honestly quite hard to say. I struggle to believe that the Elders could be so unwilling to act that they would risk escalating the standoff with the Unfettered into a real war. Our brethren are no longer merely a disaffected rabble of veterans, camping on the one world that supported radical change. They are a truly empowered movement with consolidated support at home and sympathizers amongst the Hai at large."`

Author: They have 3 worlds *now*. ;)

			`	She glances around the room and raises an eyebrow. "This is a closed session, which is quite unusual for a full Council Assembly. This does not bode well."`
			`	As the security staff take their place, the moderator walks over to address you while the Elders are still filtering in and muttering to each other. "Now, before we begin, I would like to confirm. I understand you are able to act as a witness, and confirm that the threat carried from the Unfettered was indeed real. Is this correct?"`
			choice
				`	"That is correct."`
					goto correct
				`	"I can also testify to the human position on the matter."`
			`	The moderator pulls a strange face and says, "Right, well, I doubt that will be necessary.`
			label correct

> 			`	"Thank you, Captain. Since you have become involved as a witness, protocol dictates that the session will be held in your language, though for the sake of order normal council directions will still be in Hai. We will begin in a moment."`

Author: It's order because Hai who are less versed in your language will respond more quickly to moderator declarations in Hai.

However, it occupies the same grammatical point so it either does or does not either way. Following on from a "though" I don't believe it should have one.

			`	After everyone is set in place, the moderator abruptly starts. "Teeneep has provided a confession about her visit to the Unfettered, incitement to war, involvement of outside government in internal Hai matters, and other concerns. She has justified this by an argument of necessity, that it is essential for action to be taken to produce both a peaceful outcome for the ongoing crisis with humanity and to heal a hole in the heart of our society."`

> 			`	When the moderator refers to 'healing the hole', it is spoken with an emphasis that implies heavy quotation, and you can hear the muttering of translations and scoffs rippling out among the Elders in the room.`

Author: It's people who are hired as interpreters to provide translation. They're not referred to earlier because it felt clumsy, but you would expect them if you're holding a hearing in a foreign language.

			`	Sayari hisses with distaste and quietly says, "The framing of this is terrible, someone has already decided what outcome they'd like to get and has got in on this process early."`
			`	The moderator calls something out in Hai which ends with 'Teeneep' and Sayari translates, "The Council hears Teeneep."`
			`	Not visibly disheartened by the situation, Teeneep steps into the middle and begins. She makes her case thoroughly and passionately, but with a clearly well-practiced presentation structure, ensuring that the full background for the situation is laid out for those present. Some Elders make snide comments at various points, but the moderator shuts them down and those most skeptical settle for being merely disengaged. When she presents the Unfettered's position as an offer of assistance there is a smattering of largely disbelieving sounds, though when she speaks of their willingness to force the matter the tone of the conversation gets remarkably more tense. "If you don't act, they will. Whether that's with you, or through you," she finishes.`

> 			`	One of the more stubborn Elders stands up, heralded by the moderator. "The Unfettered have made serious attempts to push through at Wah Ki before, to no avail. More recently, they've been more trouble again, and again we have organized more than enough security to keep their disruption contained, even with the current human crisis. This threat is ridiculous, and the Unfettered would not dare, not while they are so dependent on our food and medical supplies these past years. They would most certainly sacrifice many thousands of lives in the pursuit of misguided justice for a mere handful."`

Author: It's referring to the previous sentence. The subject is "the threat" which is ridiculous because they are dependent on Hai food and medical supplies, and also *carrying out the threat* would most certainly sacrifice many thousands of lives [...] for a mere handful.

				goto next
			label willed
			action
				set haiignorance
			`	Many elders chuckle around the room. The Elder says, "Ah, our pair of old rivals, divided by Cloudfire. The embers are not yet quenched then." At this, Teeneep rolls her eyes and the Elder departs.`
				goto next
			label poignant
			action
				set haiinsight
			`	There is a quiet shuffling as some of the elders sit up a little straighter. The Elder says, "A common ground between rivals, a fearsome thing indeed."`
				goto next
			label constructive
			action
				set haiinsight
			`	A roomful of laughter engulfs your words, but many Elders appear offended by their peers' mockery. The Elder who questioned you explains, "The Elders and the Unfettered have different views of what constitutes a 'constructive discussion.' You lack the biases of a Hai. If you say it was constructive, I believe you." The tone of the room changes after that, as the laughter ends. Most Elders appear to agree with your assessment, and many who laughed look ashamed.`
			label next
			`	Sayari gives you a meaningful look and whispers, "A lot of them paid attention to that..."`

> 			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`

Author: Teeneep is the threat, she is a threat to the established order.

> 			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`

Reviewer 3: Could it be made clearer then? I never would have guessed thats what was meant here

> 			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`

Reviewer: You could tell the player that Gemnali points her finger at Teeneep before she says "This threat cannot stand", and after "Order through security". There might be another way, because I really like that line building up to the end and putting that there might make it a bit worse.

> 			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`

Author: I mean, technically it's the threat that Teeneep *represents*: a person willing to wield the Hai's internal disunity against them to coerce the Elder's into a course of action they did not themselves find acceptable.
Teeneep is the threat as shorthand for that though.

I had thought the "this one here" called out teeneep clearly enough, but I guess not.

> 			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`

Reviewer 3: `This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"` I don't see a this one here in here?
mb it was added since?

> 			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`

Author: It's earlier in the diatribe, two more sentences back, part of the framing statements.

> 			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`

Reviewer 3: ~~stop linking phrases so far way~~

> 			`	As the crescendo of her impassioned statement crests to its conclusion, there is a growing rumble of ascent and the room is suddenly filled with the angry cries of those both emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen Elders jump to their feet, but the cacophony is not so easily restrained. At your side, Sayari's expression is ashen, but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`

Reviewer 2: ```suggestion
			`	There is a growing rumble of ascent as the crescendo of her impassioned statement crests to its conclusion, when she levels a clawed finger directly at Teeneep. At this, the room is filled with the angry cries of both those emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen Elders jump to their feet, but the cacophony is not so easily restrained. At your side, Sayari's expression is ashen, but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`
```

Attempt to address comments above. "Suddenly" dropped because it seems to no longer fit as well contrasted against the "growing rumble" now being in a separate sentence. "Both" moved to make it more clear both groups are not "emboldened"

> 			`	As the crescendo of her impassioned statement crests to its conclusion, there is a growing rumble of ascent and the room is suddenly filled with the angry cries of those both emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen Elders jump to their feet, but the cacophony is not so easily restrained. At your side, Sayari's expression is ashen, but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`

Reviewer 2: ```suggestion
			`	There is a growing rumble of ascent as the crescendo of her impassioned statement crests to its conclusion, when she levels a clawed finger directly at Teeneep. At this, the room is filled with the angry cries of both those emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen Elders jump to their feet, but the cacophony is not so easily restrained. At your side, Sayari's expression is ashen, but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`
```
Attempt to address comments above. "Suddenly" dropped because it seems to no longer fit as well contrasted against the "growing rumble" now being in a separate sentence. "Both" moved to make it more clear both groups are not "emboldened"

Author: I think this construction might be smoother to read?
```suggestion
			`	There is a growing rumble of ascent as the crescendo of her impassioned statement crests to its conclusion. She levels a clawed finger directly at Teeneep and the room is filled with the angry cries of both those emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen Elders jump to their feet, but the cacophony is not so easily restrained. At your side, Sayari's expression is ashen, but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`
```

Reviewer 2: Yea, I'm good with that, too.

			`	There is a shift in the sound of the room and your attention is drawn to one Hai in particular, a particularly elderly man who struggles to his feet in a series of pained stages. The effect on the assembly is profound, though. A ripple of silence flows outward from his position, and Elders all around promptly drop back into their seats as it reaches them, leaving a scant handful who were too slow to react to have their final words trail off awkwardly. Only the last speaker remains on her feet, the moderator weakly announcing in Hai what you know to be, "The council hears Aimet."`

> 			`	"Security through secrecy..." He coughs. "You throw that in there as if it were always so Gemnali but, unlike you, I was there when it was added to the mandate of this chamber."`

Reviewer: I agree that the commas should be here. I really think more people would trip up over this when they're reading it.

			`	He peers around the room, and you realize he is blind, or nearly so. "Freedom may be achieved through peace, but peace must not be maintained at the cost of freedom. Order may not be pursued at the cost of peace, nor security at the cost of order. Always this mandate must be considered in both directions. To censure this woman, to resist action on this matter. You would have us pursue secrecy at the cost of security."`

> 			`	He coughs again, and it echoes in the silence of nearly a hundred Hai holding their breath. When he continues, his voice grows stronger and sharper. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`

Author: Our parliament has 1 "speaker" who does the same thing for 176 people.
This is why standard instructions are given in Hai "for the sake of order"

> 			`	He coughs again, and it echoes in the silence of nearly a hundred Hai holding their breath. When he continues, his voice grows stronger and sharper. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`

Reviewer 3: Our parliament sucks I didnt think these Hai would as much but mb so

> 			`	He coughs again, and it echoes in the silence of nearly a hundred Hai holding their breath. When he continues, his voice grows stronger and sharper. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`

Reviewer: > They could usually write some statements etc in advance for others to consider before the meeting but this time had no time for that? Mb the moderator could make a mention of how exceptional this meeting is?

They did say that it was a closed session, and it was unusual. The Chamber of Elders seems to have been decaying for a while if Aimet was the first to improve on it?

> 			`	He coughs again, and it echoes in the silence of nearly a hundred Hai holding their breath. When he continues, his voice grows stronger and sharper. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`

Reviewer 3: Well thats the thing, whats the improvement? Closed session??

> 			`	He coughs again, and it echoes in the silence of nearly a hundred Hai holding their breath. When he continues, his voice grows stronger and sharper. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`

Reviewer 2: Bureaucracy tends to collapse under it's own weight after it is built up. Any streamlining of the process after the fact is often extraordinarily more difficult than setting the original process in motion. In the US, atm, for example, there are many Senate procedures that are explicitly not legal mandates or even precedent, but rather mere tradition, and it's often scandalous to even propose circumventing them. Getting the entire body to agree to a better alternative after thousands of years is a MASSIVE achievement.

			`	Teeneep quickly plasters back on her mask of calm as best she is able to and says, "Thank you all for your ears. You are those we trust to decide, but know this: If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed, no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`
			`	One Elder cries out, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!"`
			`	Before any further exchange can occur, though, the moderator bellows for security to escort you all from the session and shouts down the offending Elder for impropriety. The two remain engaged in a lively shouting match as you depart, the echoes following you until the chamber doors are shut.`

> 			`	Teeneep moves to pull out a datapad but both the security officers and Sayari step in to stay her hand. "The chamber will have to vote on the proposition to act," says Sayari, looking at Teeneep pointedly. "You know that external contact is forbidden during the deliberation grace window."`

Reviewer: ```suggestion
			`	Teeneep moves to pull out a datapad, but both the security officers and Sayari step in to stay her hand. "The chamber will have to vote on the proposition to act," says Sayari, looking at Teeneep pointedly. "You know that external contact is forbidden during the deliberation grace window."`
```

			choice
				`	"The deliberation grace window?"`
				`	"So what, do we just wait?"`
					goto wait
			`	Teeneep sighs and points an exasperated expression your way. "Yes, after certain types of chamber hearings there is an enforced period where the petitioner must remain isolated to the chamber's waiting area, including digitally. I forgot for a moment."`
			label wait
			`	Sayari smiles at you. "Don't worry though, it's only for about twenty-three of your minutes." She eyes the security officer warily. "After a session such as that though, the traditional wisdom is to wait in relative silence."`

> 			`	"If it wasn't a closed session we wouldn't be doing this," harrumphs Teeneep. "I hadn't realized it would be so I didn't plan for it."`

Reviewer 2: ```suggestion
			`	"If it wasn't a closed session we wouldn't be doing this," harrumphs Teeneep. "I hadn't realized it would be, so I didn't plan for it."`
```
Adding that second "I" means this now needs a comma. ;)

Author: Counteroffer:
```suggestion
			`	"If it wasn't a closed session we wouldn't be doing this," harrumphs Teeneep. "I hadn't realized it would be so didn't plan for it."`
```

Reviewer 2: That works as well, yea.

> 			`	"If it wasn't a closed session we wouldn't be doing this," harrumphs Teeneep. "I hadn't realized it would be and so hadn't planned for it."`

Reviewer: ```suggestion
			`	"If it wasn't a closed session we wouldn't be doing this," harrumphs Teeneep. "I hadn't realized it would be, I would've planned for it otherwise."`
```

Author: Doesn't feel very smooth to say aloud.

Reviewer: My suggestion sounds smooth to me, but I think it doesn't fit now. But the original doesn't work, it sounds like plain written text instead of speech.

Author: In what way?

Reviewer: "I hadn't realised it would be **and so** hadn't planned for it." It sounds like she's just read it out loud. It might be more natural like this: "I hadn't realised it would be so I didn't plan for it."

Author: ```suggestion
			`	"If it wasn't a closed session we wouldn't be doing this," harrumphs Teeneep. "I hadn't realized it would be so I didn't plan for it."`
```

			`	You're still finishing up a little shortly after they leave, at which point a harried-looking Sayari appears and practically sprints across the room to you.`
				goto alarm
			label dinner
			action
				set "learned of elders superstition"
			`	You join Teeneep and Sayari at a fusion restaurant about a block down the road from the 'Elders' Chambers' - which turns out to be the official name of the whole government complex.`
			`	Having finished your main meal, and while waiting for some dessert that Teeneep has ordered, Sayari pulls out her datapad. "You know, while we're waiting. It's probably a good idea to check the damage. I don't know if your plan is actually going to have the desired effect, you know. Especially after that scene in there; no doubt the long tooths will get riled up."`
			choice
				`	"What was that in there about the Mythos?"`
				`	"What will happen if they get riled up?"`
					goto riled

> 			`	Teeneep rolls her eyes, putting her drink down with a sharp clink. "It's superstition. Humans have their religions, the Hai have the Mythos. It's a collection of stories about Hai who lived incredibly long lives and were involved in or responsible for incredibly influential events in our history, or more accurately pre-history for the vast majority of them. If you believe the stories literally these Hai transcended mortal limitations and achieved immortality; our word for them translates imprecisely to either 'immortals' or 'imperishables'. When someone embodies a character from it well enough there’s a set of people who will believe it’s actually that person come back to steer things.`

Reviewer: ```suggestion
			`	Teeneep rolls her eyes, putting her drink down with a sharp clink. "It's superstition. Humans have their religions, the Hai have the Mythos. It's a bunch of tales about Hai who lived incredibly long lives and were involved in or responsible for incredibly influential events in our history, or more accurately pre-history, for the vast majority of them. If you believe the stories literally, these Hai transcended mortal limitations and achieved immortality; our word for them translates imprecisely to either 'immortals' or 'imperishables'. When someone embodies a character from it well enough, there’s a set of people who will believe it’s actually that person who has come back to steer things.`
```
Imperishable foodstuffs? 

I suppose Teeneep doesn't give a damn for this stuff so I want to make it more clear. I don't remember if 'whose' can be a replacement for 'who has' but it might be like 'might of' and 'might have'. It might not even need to be there... Bloody English.

Author: I mean, "who has" is entirely superfluous here, why do we need to add it?

Also, there's a difference between "not giving a damn" and actively denigrating something. "A bunch of tales" is the latter.

Reviewer: Well, if I didn't give a damn about religion, I'd call it a bunch of tales... but I can change the text. Teeneep looks like she doesn't like it here, though, if she rolls her eyes. 

Stories aren't always collected or make sense together anyway. Genesis, for example... 

```suggestion
			`	Teeneep rolls her eyes, putting her drink down with a sharp clink. "It's superstition. Humans have their religions, the Hai have the Mythos. It's a bunch of stories about Hai who lived incredibly long lives and were involved in or responsible for incredibly influential events in our history, or more accurately pre-history for the vast majority of them. If you believe the stories literally, these Hai transcended mortal limitations and achieved immortality; our word for them translates imprecisely to either 'immortals' or 'imperishables'. When someone embodies a character from it well enough there’s a set of people who will believe it’s actually that person come back to steer things.`
```

Author: Would you do so if your best friend and mentor of several decades, who would not like that language, were sitting right next to you?

Reviewer: I didn't know Sayari didn't like that language. She didn't say anything about the language, she only responds that more important people do care than Teeneep thinks. I don't think the reverend of my college wouldn't be offended if I called the bible a bunch of stories (certainly not the previous one anyway).

Author: Yeah I'm still not down with putting in "bunch" in here, but two of those commas are necessary.
```suggestion
			`	Teeneep rolls her eyes, putting her drink down with a sharp clink. "It's superstition. Humans have their religions, the Hai have the Mythos. It's a collection of stories about Hai who lived incredibly long lives and were involved in or responsible for incredibly influential events in our history, or more accurately pre-history for the vast majority of them. If you believe the stories literally, these Hai transcended mortal limitations and achieved immortality; our word for them translates imprecisely to either 'immortals' or 'imperishables'. When someone embodies a character from it well enough, there’s a set of people who will believe it’s actually that person who has come back to steer things.`
```

			`	"Someone in there has linked that scene to a story, in which I am implicated as Eyrisen."`
			label riled

> 			`	She scoffs and waves a paw dismissively. "The contingencies shouldn't be affected. Superstition is mostly dead, it's really only the Elders that subscribe to it." She pauses and looks at Sayari. "The plan will work just fine."`
> 			`	Sayari grumbles, "There's plenty of older Hai, the long tooths Captain, that subscribe to particular political factions. They tend to also subscribe, at least selectively, to the superstitions. Despite what Teeneep says, I would not count out the potential impact that scene in there could have on the situation."`
> 			`	She rounds on Teeneep, "I mean, seriously, Tee, what will you do if the plan doesn't play out fine? Will the Hai actually have to shed our brethren's blood with lethal intent?"`

Reviewer: ```suggestion
			`	She scoffs and waves a paw dismissively. "The contingencies shouldn't be affected. Superstition is mostly dead, it's really only the Elders that believe in it." She pauses and looks at Sayari. "The plan will work just fine."`
			`	Sayari grumbles, "There's plenty of older Hai, those 'long tooths', that align to particular political factions. They also tend to sympathize, at least selectively, to the superstitions. Despite what Teeneep says, I would not count out the potential impact that scene in there could have on the situation."`
			`	She rounds on Teeneep, "I mean, seriously, Tee, what will you do if the plan doesn't play out fine? Will the Hai actually have to shed our brethren's blood with lethal intent?"`
```

Author: The change to "align" is fine, but "subscribe" is intentional language for all occurrences referring to the superstition.

Reviewer: Yeah, they work, but I think 'subscribe' would be used too much here. 

```suggestion
			`	She scoffs and waves a paw dismissively. "The contingencies shouldn't be affected. Superstition is mostly dead, it's really only the Elders that subscribe to it." She pauses and looks at Sayari. "The plan will work just fine."`
			`	Sayari grumbles, "There's plenty of older Hai, those 'long tooths', that align with particular political factions. They also tend to subscribe, at least selectively, to the superstitions. Despite what Teeneep says, I would not count out the potential impact that scene in there could have on the situation."`
			`	She rounds on Teeneep. "I mean, seriously, Tee, what will you do if the plan doesn't play out fine? Will the Hai actually have to shed our brethren's blood with lethal intent?"`
```
Round means 'to turn'? Then there has to be a full stop if round doesn't meant 'to speak'.
